wscl-issues/proposed/go-error

```diff
@@ -0,0 +1,118 @@
+Issue:          GO-ERROR
```

Comment:
I would prefer a name like `GO-NON-EXISTENT-TAG` since there could be other errors such as `GO-ABANDONED-EXIT-POINT`.

Reply:
Fixed